units/src/block.rs

```diff
@@ -6,7 +6,7 @@
 //!
 //! These are general types for abstracting over block heights, they are not designed to use with
 //! lock times. If you are creating lock times you should be using the
-//! [`locktime::absolute::Height`] and [`locktime::relative::Height`] types.
+//! [`locktime::absolute::Height`](crate::locktime) and [`locktime::relative::Height`](crate::locktime) types.
```

Comment:
This `(crate::locktime)` shouldn't be needed, we bring `locktime` into scope already using `#[cfg(doc)]`.

Reply:
Thanks, I see this now and have removed it.  I was running `cargo doc` locally to check the document pages and the link didn't work, I assume it is something obvious I am not doing like adding a needed option.

Also `pub struct BlockHeight` and `BlockInterval` don't appear on the `docs.rs` page so I couldn't check there.  The reason for this may also be obvious but why is that?

Reply:
You might be missing the `RUSTDOCFLAGS` here https://github.com/rust-bitcoin/rust-bitcoin-maintainer-tools/blob/3494ceec521cbc2cbe7e4b0d3fb9219a2164f57d/ci/run_task.sh#L277

@tcharding do you think it useful if we add a `just` command to build the docs?

```bash
just docs
```

Reply:
Thanks, that fixed it

Reply:
I thought we added that `just` command but it seems not. I think we should, I personally use a shell alias `RUSTDOCFLAGS="--cfg docsrs" cargo +nightly rustdoc --all-features -- -D rustdoc::broken-intra-doc-links
`

Reply:
I  created https://github.com/rust-bitcoin/rust-bitcoin/issues/2965